### PR TITLE
Add fix for Wrapping exposed

### PR DESCRIPTION
**Pull request by diegoteran**
Fixed #4494

### CODE REVIEW

File: packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {TextureFilter, Wrapping} from 'three';
+import {TextureFilter} from 'three';
```
Comment (elalish) [inline packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts:1]: We need to remove this too.

**Review by elalish — CHANGES_REQUESTED** [review]
Can you update your Transform Textures example to demonstrate this is working now? Just add a little drop-down for the wrap mode and apply same to both axes.

File: packages/modelviewer.dev/examples/scenegraph/index.html [hunk 2]
```diff
@@ -499,6 +499,12 @@ <h2 class="demo-title">Transform textures</h2>
       <input type="range" min="0.5" max="1.5" value="1" step="0.01" id="scaleSlider">
       <p>Offset</p>
       <input type="range" min="0" max="1" value="0" step="0.01" id="offsetSlider">
+      <p>WrapMode</p>
+      <select id="wrapMode">
+        <option value="10497">Repeat</option>
```
Comment (elalish) [inline packages/modelviewer.dev/examples/scenegraph/index.html:504]: Can you switch this to `Wrap.Repeat` etc? I want to ensure our enum is properly exported.

Comment (elalish) [reply packages/modelviewer.dev/examples/scenegraph/index.html:504]: If it doesn't work, we probably have to re-export it from model-viewer.ts.

**Review by elalish — APPROVED** [review]
Thanks

File: packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts (hunk 1 above)
Comment (diegoteran) [reply packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts:1]: Leaving this for a follow up PR.

File: packages/modelviewer.dev/examples/scenegraph/index.html (hunk 2 above)
Comment (diegoteran) [reply packages/modelviewer.dev/examples/scenegraph/index.html:504]: We realized at the office that making this work might take a bit more time of work, leaving it for another index.html cleanup :/

**Pull request merged**